.github/workflows/tarball.yml

```
@@ -61,47 +61,47 @@ jobs:
         with:
           name: tarball-testing
       - name: Expand
-        run:  tar xfz omniperf-*.tar.gz; rm omniperf-*.tar.gz
+        run:  tar xfz rocprof-compute-*.tar.gz; rm rocprof-compute-*.tar.gz
       - name: Python dependency installs
         run: |
-          cd omniperf-*
+          cd rocprof-compute-*
           python3 -m pip install -t${INSTALL_DIR}/python-libs -r requirements.txt
       - name: Configure
         run:  |
-          cd omniperf-*
+          cd rocprof-compute-*
           mkdir build
           cd build
-          cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR}/omniperf \
+          cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR}/rocprof-compute \
             -DPYTHON_DEPS=${INSTALL_DIR}/python-libs ..
       - name: Install
         run:  |
-          cd omniperf-*
+          cd rocprof-compute-*
           cd build
           make install
       - name: Verify expected paths
         run: |
           # find $INSTALL_DIR
-          test -d $INSTALL_DIR/omniperf
-          test -x $INSTALL_DIR/omniperf/bin/omniperf
-          test -s $INSTALL_DIR/omniperf/libexec/omniperf/VERSION
-          test -s $INSTALL_DIR/omniperf/libexec/omniperf/VERSION.sha
-          test -d $INSTALL_DIR/omniperf/libexec/omniperf/omniperf_analyze
-          test -d $INSTALL_DIR/omniperf/libexec/omniperf/omniperf_profile
-          test -d $INSTALL_DIR/omniperf/libexec/omniperf/omniperf_soc
-          test -d $INSTALL_DIR/omniperf/libexec/omniperf/utils
-          test -s $INSTALL_DIR/omniperf/share/omniperf/sample/vcopy.cpp
-          test -d $INSTALL_DIR/omniperf/share/omniperf/modulefiles
+          test -d $INSTALL_DIR/rocprof-compute
+          test -x $INSTALL_DIR/rocprof-compute/bin/rocprof-compute
+          test -s $INSTALL_DIR/rocprof-compute/libexec/rocprof-compute/VERSION
+          test -s $INSTALL_DIR/rocprof-compute/libexec/rocprof-compute/VERSION.sha
+          test -d $INSTALL_DIR/rocprof-compute/libexec/rocprof-compute/rocprof_compute_analyze
+          test -d $INSTALL_DIR/rocprof-compute/libexec/rocprof-compute/rocprof_compute_profile
+          test -d $INSTALL_DIR/rocprof-compute/libexec/rocprof-compute/rocprof_compute_soc
+          test -d $INSTALL_DIR/rocprof-compute/libexec/rocprof-compute/utils
+          test -s $INSTALL_DIR/rocprof-compute/share/rocprof-compute/sample/vcopy.cpp
+          test -d $INSTALL_DIR/rocprof-compute/share/rocprof-compute/modulefiles
```

```suggestion
          test -d $INSTALL_DIR/rocprof-compute
          test -x $INSTALL_DIR/rocprof-compute/bin/rocprofiler-compute
          test -s $INSTALL_DIR/rocprof-compute/libexec/rocprofiler-compute/VERSION
          test -s $INSTALL_DIR/rocprof-compute/libexec/rocprofiler-compute/VERSION.sha
          test -d $INSTALL_DIR/rocprof-compute/libexec/rocprofiler-compute/rocprof_compute_analyze
          test -d $INSTALL_DIR/rocprof-compute/libexec/rocprofiler-compute/rocprof_compute_profile
          test -d $INSTALL_DIR/rocprof-compute/libexec/rocprofiler-compute/rocprof_compute_soc
          test -d $INSTALL_DIR/rocprof-compute/libexec/rocprofiler-compute/utils
          test -s $INSTALL_DIR/rocprof-compute/share/rocprofiler-compute/sample/vcopy.cpp
          test -d $INSTALL_DIR/rocprof-compute/share/rocprofiler-compute/modulefiles
```
This subfolder will match the package name, rocprofiler-compute